### PR TITLE
Add watermark

### LINKED ISSUE #18: Add watermark / button to bottom left of the screen
Add a watermark to the bottom right corner of the screen as discussed in dev-chat.
Ideally this would act as a button for the menu of the app, and should be removable by the user if desired.

Assets attached -> [digiview-feed-overlays.zip](https://github.com/fpvout/DigiView-Android/files/6481304/digiview-feed-overlays.zip)
I've added images with and without the text as this may want to be localized in the future.

examples:
digiview feed#1
digiview feed#5

### PR DESCRIPTION
**Pull request by fichek**
Excuse my messy desk
* resolves #18 by adding the watermark with a subtle shadow so it doesn't disappear completely on light video feeds
* added a fade in/out animation to hide/show the watermark when tapping anywhere
* added font from #24 to be used in the watermark in case we ever need to localize "feed"
* added styles.xml and added a style to be used with the watermark (will be adding more styles in subsequent PRs so not closing that yet)
* changed app orientation mode to `sensorLandscape` to be able to use both landscape orientations (we should decide what to do with #20 but felt this was needed at least for now)

### CODE REVIEW

File: app/src/main/java/com/fpvout/digiview/MainActivity.java [hunk 1]
```diff
@@ -72,6 +81,37 @@ protected void onCreate(Bundle savedInstanceState) {
...
+
+    private void toggleWatermark() {
+        if (watermarkAnimationInProgress) {
+            return;
+        }
+        watermarkAnimationInProgress = true;
+
+        float targetAlpha = 0;
+        if (watermarkView.getAlpha() == 0) {
+            targetAlpha = 1;
```
Comment (omouren) [inline app/src/main/java/com/fpvout/digiview/MainActivity.java]: Set alpha to the same in the layout : 0.3F

**Review by omouren — CHANGES_REQUESTED** [review]

**Comment by omouren** [comment]
I like it!
Just the logo is not turning itself to the same transparency when we touch the screen

And the image is not perfectly centered with the text (check your image file, we can see that the goggles are slightly to the left)

**Comment by fichek** [comment]
> I like it!
> Just the logo is not turning itself to the same transparency when we touch the screen
> 
> And the image is not perfectly centered with the text (check your image file, we can see that the goggles are slightly to the left)

Fixed both. Made so many versions of these icons until I was happy with the shadow, dunno how I missed the alignment, thanks 😄

**Review by omouren — APPROVED** [review]

**Review by jlucidar — APPROVED** [review]
great !

**Pull request merged**